recipes/green/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.11.0" %}
+
+package:
+  name: greem
+  version: {{ version }}
+
+source:
+  fn: green-{{ version }}.tar.gz
+  url:   https://github.com/CleanCut/green/archive/{{version}}.tar.gz
+  sha256: fb78e51a15872c6a4f2952fc6857c91fef390d89b1d725d85b609c119584c137
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    
+test:
+  imports:
+    - green
+
+about:
+  home: https://github.com/CleanCut/green
+  license: MIT
+  license_file: LICENSE
+  summary: 'Green is a clean, colorful, fast python test runner'
+
```

Comment:
You might as well add `description`, `doc_url`, and `dev_url` too, like in [the example](https://github.com/conda-forge/staged-recipes/blob/ff0002dc6f5ff8a580ca0f58faaa9bec7c44df93/recipes/example/meta.yaml#L62).